src/functions/config.ts

```diff
@@ -201,6 +202,30 @@ class Config {
     return this.config.builderApiUrl.replace('{HOST}', this.host);
   }
 
+  public additionalHeaders(): Record<string, string> {
+    let tenantId = null;
+
+    if (this.config.tenantId) {
+      tenantId = this.config.tenantId;
+    } else {
+      switch (this.zone) {
+        case 'edge':
+          tenantId = '62313937-6230-3361-3465-643833656463';
+          break;
+        default:
+          tenantId = null;
+      }
+    }
+
+    if (tenantId) {
+      return {
+        'X-FusionAuth-TenantId': tenantId,
+      };
+    }
+
+    return {};
+  }
```

Comment:
I think it's better to move this function to `utils/login.ts` as it's only used for loggin in. In `config.ts` I'd expect a function like
```
get tenantId(): string {
  return this.config.tenantId;
};
```